scripts/travis/build.sh

```diff
@@ -9,6 +9,11 @@ set -e
 DIR=`dirname $0`
 source ${DIR}/common.sh
 
+# check if the ci environ argument is provided
+if [ $# -eq 0 ]; then
+  echo "ci environ arg not provided (travis|applatix)"
```

Comment:
if the argument is not valid, we exit right away - since it is a mandatory argument